cloud_tracking.py

```diff
@@ -234,43 +268,29 @@ def send_coverage(self, coverage):
         post_id = posts.insert_one(post).inserted_id
         print("post_id: " + str(post_id))
 
-    def send_cloud(self, frame):
-        """Sends cloud image to database"""
-        success, im_buffer = cv2.imencode('.png', frame)
+    def send_image(self, image, event_name):
+        if send_images is False or sock is None:
+            return
+        success, im_buffer = cv2.imencode('.png', image)
 
         if success is False:
             print("couldnt encode png image")
             return
 
         byte_image = im_buffer.tobytes()
-        post = {
-            "author": "cloud_tracking.py",
-            "coverage": byte_image
-        }
+        sock.emit(event_name, byte_image)
 
-        posts = self.db.cloudImage
-        post_id = posts.insert_one(post).inserted_id
-        print("post_id: " + str(post_id))
+    def send_cloud(self, frame):
+        """Sends cloud image to website via socketIO"""
+        send_image(frame, 'coverage')
 
     def send_shadow(self, coverage):
-        """Sends shadow image to database"""
+        """Sends shadow image to website via socketIO"""
```

Comment:
The comment says sending to website but the code is sending to the database. Which one is it doing?

Reply:
umm no... read send_image... has sock.emit... so it is sending to the website

Reply:
Oh whoops I was looking at the code that got deleted. Didn't even notice it was highlighted red. Sorry about that